osu.Game/Rulesets/Objects/Types/ComboIndex.cs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+
+namespace osu.Game.Rulesets.Objects.Types
+{
+    /// <summary>
+    /// Represents an <see cref="Ordinal"/> and <see cref="WithOffset"/> combo indices.
```

Comment:
this isn't english nor does it describe the class. don't describe something using itself.

Reply:
I've reworded it, please check if it's good now.